Sources/Foundation/FileManager.swift

```diff
@@ -1149,8 +1149,9 @@ open class FileManager : NSObject {
     
     open func homeDirectory(forUser userName: String) -> URL? {
         guard !userName.isEmpty else { return nil }
-        guard let url = CFCopyHomeDirectoryURLForUser(userName._cfObject) else { return nil }
-        return  url.takeRetainedValue()._swiftObject
+        // Prefer to take the `CFCopyHomeDirectoryURLForUser` path for the
+        // current user.
+        return CFCopyHomeDirectoryURLForUser(userName == NSUserName() ? nil : userName._cfObject)?.takeRetainedValue()._swiftObject
```

Comment:
This ternary is hurting my eyes. It's too hard to see where the parameter to `CFCopyHomeDirectoryURLForUser` ends and where the `?.takeRetainedValues()...` begins.

Reply:
Spoke with @etcwilde offline, reflowed text seems to address the concern.